src/components/hero/Hero.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { HeroText } from './HeroText';
+
+interface Props {
+  heroText: string;
+  listLength: number;
+}
+
+export const Hero = ({ heroText, listLength }: Props) => {
+  return (
+    <>
+      <Layout>
+        <HeroText text={heroText} listLength={listLength} />
```

Comment:
```suggestion
export const Hero = ({ text, listLength }: Props) => {
  return (
    <>
      <Layout>
        <HeroText text={text} listLength={listLength} />
```

hero가 불필요한 prefix인것 같아서요.

Reply:
넵! 수정해보겠습니다